src/NuGet.Clients/PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -67,17 +67,14 @@ public virtual void CleanUp()
         /// <param name="filter">The current filter. This will used to select the default action.</param>
         public async virtual Task SetCurrentPackage(
             PackageItemListViewModel searchResultPackage,
-            ItemFilter filter)
+            ItemFilter filter,
+            InfiniteScrollList packageList)
```

Comment:
Don't pass the entire control to a model. The only bit needed here is `SelectedItem`.